lib/game.js

```diff
@@ -193,12 +194,11 @@ Game.prototype.drawHealth = function(context) {
 };
 
 Game.prototype.drawStart = function(context){
-  if (this.gameOver === true) {
+  if (this.gameOver) {
     this.drawLastScore(context);
-  }
-  if (this.gameOver === false){
+  } else {
```

Comment:
I had tried this, but for some reason it wasn't working for me unless i explicitly said true and false! weird.